### ISSUE TITLE
No feedback from `flutter run` when targeting iOS hardware, until "list index out of range"

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#6708]
My iPhone is plugged in, and I ran `flutter run`. I don't see anything in response, while the tool is running xcodebuild and the other steps.

Then, after many seconds, I see this:

```
~/Code/flutter/examples/flutter_gallery[master*] $ flutter run
Traceback (most recent call last):
  File "/tmp/fruitstrap_d83d5bc53967baa0ee18626ba87b6254b2ab5418.py", line 35, in run_command
    lldb.target.Launch(lldb.SBLaunchInfo(shlex.split(args[1] and args[1] or '--enable-checked-mode')), error)
IndexError: list index out of range
^C
```

1. Can we display some sort of spinner/hint that the tool is working in the background?
2. Is the error important?

**Comment by sethladd** [comment]
cc @cbracken who was looking at this with me.

**Comment by jason-simmons** [comment]
The missing log output may be fixed by https://github.com/flutter/flutter/pull/6720

**Comment by cbracken** [comment]
Fixed by #7794
